app/controllers/learning_hours_controller.rb

```diff
@@ -55,6 +55,8 @@ def destroy
 
   def set_learning_hour
     @learning_hour = LearningHour.find(params[:id])
+  rescue ActiveRecord::RecordNotFound
+    redirect_to learning_hours_path
```

Comment:
hmm someday I want to unify our error handling, but not in this PR. Something like - 

```
class BaseController < ActionController::API
    include Pundit
    before_action :doorkeeper_authorize!
    before_action :validate_user!
    after_action :verify_authorized
    after_action :verify_policy_scoped, only: :index

    rescue_from ActiveRecord::RecordNotFound, with: :render_not_found
```